### ISSUE TITLE
html sections

### ISSUE THREAD

**Issue by rimmesbe** [gary9630/prj-rev-bwfs-dasmoto#1]
See how you have naturally split your html into sections, take it to the next level and wrap those elements in their own div. Then move the class to the div to act as a section label, while still being able to uniquely target the h2's:
.brushes h2 { }
<img width="122" alt="screen shot 2017-06-28 at 1 57 34 pm" src="https://user-images.githubusercontent.com/9957628/30193605-0f4fabd4-941c-11e7-9331-51e2f9290c7c.png">
https://github.com/gary9630/prj-rev-bwfs-dasmoto/blob/master/Unit2_Dasmoto/index.html#L11-L14

**Comment by gary9630** [comment]
Yeah, I've considered to add div when I writing this part. Is it more beneficial when the codes grown larger by adding div? Thanks for your review.
